### ISSUE TITLE
Make it clear you still have to do `mocha test.js`

### ISSUE THREAD

**Issue by domenic** [Joris-van-der-Wel/mocha-sugar-free#1]
I thought that since my tests didn't use the Mocha globals, I could just do `node test.js`. Seems like that's not the case.

**Comment by Joris-van-der-Wel** [comment]
Good point. I'll add a note. 

It would probably be possible to support what you tried to do. Do you think this would be worthwhile?

**Comment by domenic** [comment]
Yeah, it would definitely save some headache on my current (non-jsdom) project. I need to muck with require.extensions _before_ kicking off Mocha. Currently my best bet appears to be starting Mocha programmatically, which is annoying. I thought mocha-sugar-free would solve my problems by letting me just do `require("./test.js")`.

**Comment by steambap** [comment]
Occasionally I want to run my mocha test with debugger, if I can just do `node test.js`, it will be very helpful.
